hibernate4/src/main/java/com/fasterxml/jackson/datatype/hibernate4/Hibernate4Module.java

```diff
@@ -78,16 +78,22 @@
          * @since 2.8.2
          */
         REPLACE_PERSISTENT_COLLECTIONS(false),
-        
+
         /**
          * Using {@link #FORCE_LAZY_LOADING} may result in
          * `javax.persistence.EntityNotFoundException`. This flag configures Jackson to
          * ignore the error and serialize a `null`.
          *
          * @since 2.10
          */
-        WRITE_MISSING_ENTITIES_AS_NULL(false)
-        ;
+        WRITE_MISSING_ENTITIES_AS_NULL(false),
+
+        /**
+         * Feature that may be disables to unwrap the identifier
+         * of the serialized entity, returning a value instead of
+         * an object.
+         */
```

Comment:
Could you add "@since 2.12" javadoc annotation at the end of javadoc comment here? (and same for hibernate5)